lib/rihanna/job.ex

```diff
@@ -59,6 +60,15 @@ defmodule Rihanna.Job do
   end
   ```
 
+  You can define a `priority/0` function which will be called if no priority
+  is set when a job is enqueued. It should return a single integer. A priority
+  of 1 will be the highest priority, with larger numbers being run after.
```

Comment:
Surely nothing prevents you from setting a 0 or negative priority?

Reply:
Nothing prevents it. I was debating trying to make it positive-only.

Reply:
If it follows the linux scale, negative numbers should be fine.